java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -1280,4 +1281,33 @@ public CompletableFuture<Long> bitcount(
                 new String[] {key, Long.toString(start), Long.toString(end), options.toString()};
         return commandManager.submitNewCommand(Bitcount, arguments, this::handleLongResponse);
     }
+
+    @Override
+    public CompletableFuture<Long> bitpos(@NonNull String key, long bit) {
```

Comment:
Why are we using `long` to specify a single bit?  
Should we be returning Long or [Byte](https://docs.oracle.com/javase/8/docs/api/java/lang/Byte.html)? 
```suggestion
    public CompletableFuture<Long> bitpos(@NonNull String key, byte bit) {
```